predictsignauxfaibles/explain.py

```diff
@@ -0,0 +1,167 @@
+from types import ModuleType
+
+import numpy as np
+import pandas as pd
+
+from predictsignauxfaibles.data import SFDataset
+from predictsignauxfaibles.utils import (
+    sigmoid,
+    make_multi_columns,
+)
+
+
+def list_concerning_contributions(entry: pd.Series, thr=0.07):
```

Comment:
Why is 0.07 the default ? Do we _need_ a default value for `thr` here ?

Reply:
Default threshold 0.07 was selected through a first manual fit on examples where the contribution of groups of variables seemed obvious. The threshold will probably be adjusted though, which suggests that we should add a CLI option to specify a threshold. To follow based on feedback from the front-end team!